app/assets/stylesheets/base/_fonts.scss

```diff
@@ -14,25 +14,7 @@
 
 @font-face {
```

Comment:
It might be good to include Aeonik Bold web font too, just in case there are any instances where a title is bolded.

While Medium is our primary brand font, there could be some instances of `font-weight: bold;` in the stylesheet codebase set on elements with Aeonik. We could remove any bold font rules where Aeonik is in place—if we don’t browsers apply a faux bold style that generally looks quite bad, [especially in Safari](https://alistapart.com/article/say-no-to-faux-bold/).

Supplying the bold web font should avoid this edge case—even if we prefer medium.

Reply:
@BrettJay good point. I have mused about this and I went with the direction of loading as little asset as possible. 

That being said, @mbelton-buildkite can you think of any instance when we might use `<strong>` within a heading? I haven't spotted any, but if there's a chance of using this then we can load the Aeonik Bold font.

Quick Figma mockup of what it looks like:

![image](https://github.com/buildkite/docs/assets/7202667/046ee076-39f7-45ca-b8c5-40fb75eac41c)